x-pack/plugins/security_solution/public/cases/components/all_cases/index.tsx

```diff
@@ -322,14 +322,40 @@ export const AllCases = React.memo<AllCasesProps>(
     const isDataEmpty = useMemo(() => data.total === 0, [data]);
 
     const TableWrap = useMemo(() => (isModal ? 'span' : Panel), [isModal]);
+
+    const tableRowProps = useCallback((item) => {
+      const rowProps = {
+        'data-test-subj': `cases-table-row-${item.id}`,
+      };
+
+      if (isModal) {
+        return {
+          ...rowProps,
+          onClick: () => {
+            if (onRowClick != null) {
+              onRowClick(item.id);
```

Comment:
don't we need to put `onRowClick` to `useCallback` dependencies?

Reply:
You are right. The `/* eslint-disable react-hooks/exhaustive-deps */` at the beginning of the file made me to miss it.